requirements-min.txt

```diff
@@ -3,3 +3,4 @@ zarr==2.11.0
 numcodecs==0.9.1
 pynwb==2.0.0
 setuptools
+numpy==1.19.3
```

Comment:
Could you please add a comment describing why this is the minimum version?

Also if this is the minimum version, it should also be specified in the `reqs` variable in `setup.py`